src/test/java/seedu/rc4hdb/logic/commands/modelcommands/EditCommandTest.java

```diff
@@ -18,7 +18,7 @@
 
 import seedu.rc4hdb.commons.core.Messages;
 import seedu.rc4hdb.commons.core.index.Index;
-import seedu.rc4hdb.logic.commands.modelcommands.EditCommand.EditPersonDescriptor;
+import seedu.rc4hdb.logic.commands.modelcommands.EditCommand.EditResidentDescriptor;
```

Comment:
This nested class was removed from `EditCommand`, right? Does this test pass locally?